src/transformers/training_args.py

```diff
@@ -85,20 +85,6 @@
 trainer_log_levels = dict(**log_levels, passive=-1)
 
 
-TORCH_COMPILE_BACKENDS = [
-    "eager",
-    "aot_eager",
-    "inductor",
-    "nvfuser",
-    "aot_nvfuser",
-    "aot_cudagraphs",
-    "ofi",
-    "fx2trt",
-    "onnxrt",
-    "ipex",
-]
-
```

Comment:
`torch._dynamo.list_backends()` is supposed to provide the supported back-ends. So we could use this API as well.

Reply:
Indeed, but since we don't check the vlaues for now, we have no need for it.